app/src/main/java/org/fossasia/openevent/general/auth/EditProfileViewModel.kt

```diff
@@ -0,0 +1,57 @@
+package org.fossasia.openevent.general.auth
+
+import android.arch.lifecycle.MutableLiveData
+import android.arch.lifecycle.ViewModel
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.schedulers.Schedulers
+import org.fossasia.openevent.general.utils.JWTUtils
+import org.json.JSONException
+import timber.log.Timber
+
+class EditProfileViewModel(private val authService: AuthService, private val authHolder: AuthHolder) : ViewModel() {
```

Comment:
I think you can use only the authHolder and get the work done

Reply:
we need auth service as updateUser is in authservice

Reply:
resolved